lib/manager.js

```diff
@@ -40,66 +40,9 @@ var expandRequestUrlInput = function(sites) {
 // expectedText property out into an arry of objects
 // if it is not already an array of objects.
 var expandExpectedTextInput = function(sites) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['sites' is defined but never used.](https://www.codacy.com/app/kevin_1379/codacy/file/1533868191/issues/source?bid=3007997&fileBranchId=3010243#l42)